frontend/src/components/sample.txt

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState } from 'react';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { useForm } from 'react-hook-form';
+import axios, { AxiosResponse, AxiosError } from 'axios';
+​
+type Article = {
+  id: Number,
+  title: string,
+  content: string,
+  created_at: string,
+};
+​
+type ArticleInput = {
+  title: string,
+  content: string,
+};
+​
+const articleValidation = { 
+  title: {
+    required: {
+      value: true,
+      message: 'Title field is required.'
+    },
+    max: {
+      value: 255,
+      message: 'Title field max character up to 255 only.'
+    }
+  },
+  content: {
+    required: {
+      value: true,
+      message: 'Content field is required.',
+    },
```

Comment:
Please remove the whole file since its not part of the project. You can keep this in your local directory instead but no need to include in the project

Reply:
done